apps/web/components/Organization/Wrapper/Themes/SyslumennTheme/SyslumennFooter.tsx

```diff
@@ -22,14 +23,23 @@ interface FooterProps {
   title: string
   logo?: string
   footerItems: Array<FooterItem>
+  questionsAndAnswersText?: string
+  canWeHelpText?: string
 }
 
 export const SyslumennFooter: React.FC<FooterProps> = ({
   title,
   logo,
   footerItems,
+  questionsAndAnswersText = 'Spurningar og svör',
+  canWeHelpText = 'Getum við aðstoðað?',
```

Comment:
i18n?

Reply:
![image](https://user-images.githubusercontent.com/43557895/168095558-090ccf5b-f734-4f73-af3f-4b014d855ec8.png)
I am using translations, these are just default values if you do not provide the props

Reply:
Right, my bad. Good job :)